index.d.ts

```diff
@@ -0,0 +1,18 @@
+export class Claim {
+	getGuid(): string;
+    getMainSnak(): Snak;
```

Comment:
Spacing seems weird here

Reply:
whoops, that'll teach me to change IDE halfway through

Reply:
I think this is fixed now right